O/open62541/open62541@1.3/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "open62541"
+version = v"1.3.10"
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/open62541/open62541.git",
+              "2405c6588f8d8093e3e0e49e920db64daf479799")
+]
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/open62541/
+mkdir build && cd build/
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DUA_MULTITHREADING=100 \
+    -DUA_ENABLE_SUBSCRIPTIONS=ON \
+    -DUA_ENABLE_METHODCALLS=ON \
+    -DUA_ENABLE_PARSING=ON \
+    -DUA_ENABLE_NODEMANAGEMENT=ON \
+    -DUA_ENABLE_IMMUTABLE_NODES=ON \
+    -DUA_ENABLE_HISTORIZING=ON \
+    -DBUILD_SHARED_LIBS=ON \
+    -DUA_FORCE_WERROR=OFF \
+    ..
+make -j${nproc}
+make install
+install_license ../LICENSE
+"""
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```

Comment:
`experimental` does nothing
```suggestion
platforms = supported_platforms()
```